tutorials/load-balancing-with-varnish.md

```diff
@@ -0,0 +1,91 @@
+# Load balancing with Varnish
+
+If you are using npm Enterprise and want to apply Varnish as a solution for load balancing then follow the steps below.
+
+* Install and configure nginx.
+
+    * [Configure nginx](https://gist.github.com/bcoe/0f2ca5e644d464f40f9937c9787a244c)
+    * [Increase limits](https://www.masv.io/boost-nginx-connection-limits/)
+
+* [Install and configure varnish.](https://varnish-cache.org/docs/trunk/installation/install.html?highlight=installation)
+
+Open `*.vcl` file (by default it is `default.vcl`).
+
+Varnish has a concept of "backend" servers. A backend server is the server providing the content that Varnish will accelerate.
+
+Somewhere in the top there will be a section that looks a bit like this:
+
+```
+backend default {
+    .host = "127.0.0.1”;
+    .port = "8080”;
+ }
+ ```
+
+This defines a backend in Varnish called `default`. When needs to get content from this backend it will connect to port 8080 on `localhost (127.0.0.1)`.
```

Comment:
Minor correction: "When Varnish serves content from this backend, it will connect to port 8080 on `127.0.0.1`."